src/zos_files/zowe/zos_files_for_zowe_sdk/datasets.py

```diff
@@ -474,32 +581,44 @@ def create_default(self, dataset_name: str, default_type: str):
         response_json = self.request_handler.perform_request("POST", custom_args, expected_code=[201])
         return response_json
 
-    def get_content(self, dataset_name, stream=False):
-        """Retrieve the contents of a given dataset.
+    def get_content(self, dataset_name: str, stream: bool = False) -> dict:
+        """
+        Retrieve the contents of a given dataset.
+
+        Parameters
+        ----------
+        dataset_name: str
+            The name of the dataset
+        stream: bool
+            Specifies whether the request is streaming
```

Comment:
Should we include the default value?
```suggestion
            Specifies whether the response is streamed. Default: False
```